.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,15 +3,17 @@
   WIP: work in progress
   🐛    Fix a bug.
   ✨    Introduce new features.
-  ♻️     Refactor code.
+  ♻️    Refactor code.
   🚑️    Critical hotfix.
-  ⚗️     Perform experiments.
-  ⬆️     Upgrade dependencies.
+  🎨    Enhance existing feature.
+  ⚗️    Perform experiments.
+  ⬆️    Upgrade dependencies.
   📝    Add or update documentation.
   🔨    Add or update development scripts.
   🔒️    Fix security issues.
-  ⚠️     Changes in devops configuration
-  🗃️    Migration of database
+  ⚠️    Changes in associated codebases: changes in ops configuration etc. required before deploying: Please link associated issue or PR
```

Comment:
these two are kind f the exception to the rule above because we have them since so long that it might be confusing changing them

Reply:
I do not understand thisone: `` Changes in associated codebases`` ?? 

I suggest `` Changes in ops configuration etc. required before deploying. Please link associated issue or PR``

Reply:
done thx